includes/parser/Parser.php

```diff
@@ -4029,9 +4029,6 @@ function fetchScaryTemplateMaybeFromCache( $url ) {
 
 		$text = Http::get( $url );
 		if ( !$text ) {
-			# wikia start
-			Wikia::log(__METHOD__, false, "Scary transclusion failed for <{$url}>");
-			# wikia end
```

Comment:
Good point. I'll compare logs from `Http::get` and the ones coming from this `Wikia::log` call .